bedrock/firefox/firefox_details.py

```diff
@@ -296,11 +296,11 @@ class FirefoxAndroid(_ProductDetails):
     }
 
     store_url = settings.GOOGLE_PLAY_FIREFOX_LINK
-    aurora_url_base = ('https://ftp.mozilla.org/pub/mozilla.org/mobile/nightly/'
+    aurora_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
                        'latest-mozilla-aurora-android')
```

Comment:
interestingly these are not concatenated. When clicking the link I get:
`https://archive.mozilla.org/pub/mobile/nightly/latest-mozilla-aurora-android-api-15/fennec-44.0a2.multi.android-arm.apk`